### DISCUSSION TITLE
Python pandas

### DISCUSSION THREAD

**Discussion by heham** [termux/termux-app#3144] [General]
安装118版termux，进去update然后安装Python，3.11版本，然后安装pandas不成功，卡numpy，查看前面的解决办法手动安装比如1.32.2不成功，最后尝试1.22.2numpy可以装上，然后安装cython和pandas还是卡在numpy，尝试过1.4到1.5版本的pandas都不行。
继续尝试proot-distro安装ubuntu，如果termux里装了Python这里面也能看到，但是还是安装pandas不成功，如果termux里没装Python直接在ubuntu里装Python-full，那这时会出现pip不能用，单独安装Python和pip一样不行

**Comment by 2096779623** [comment]
Please use english.

pandas: https://github.com/termux/termux-packages/issues/13785
